.circleci/config.yml

```diff
@@ -13,12 +13,12 @@ aliases:
 
   - &yarn-install
     run: |
-      sudo npm i -g yarn@^1.2.1
+      sudo curl -o- -L https://yarnpkg.com/install.sh | bash
```

Comment:
should we specify a version?

Reply:
This will automatically resolve the latest one. I just tested on my Windows machine

Reply:
Yes, but I don't know if we _want_ latest instead of being specific. @cpojer?

Reply:
I'll wait for confirmation 👍

Reply:
Circle has support for yarn now, no need to install if we don't need to enforce a specific version:

https://circleci.com/docs/1.0/yarn/

Reply:
shall I remove the part of installing from the script ?

Reply:
I would remove it and give it a try, I think in Circle 2.0 (which we're on) you don't need to install or override anything (similar to npm)

Reply:
These are docs for circle v1, we use v2 and whether yarn is installed is specified by docker image we use. It's already installed in circle defined images, but not necessarily on custom ones (browser testing)

Reply:
Ah, thanks @thymikee, what do you think about:

`which yarn || curl -o- -L https://yarnpkg.com/install.sh | bash`

Reply:
cool

Reply:
oh I just removed it, I suppose I can use the line from @rickhanlonii  :)

Reply:
@rickhanlonii I think the test-browser pipeline doesn't like the installing yarn by curl

Reply:
Assuming version doesn't matter we could use:

```bash
which yarn || npm install -g yarn
```

Reply:
however, @SimenB said previously it's not recommended to use npm to install yarn. Therefore he suggested me to use curl

Reply:
Looks like yarn isn't getting added to the path when installing with bash, I think we can get that working with a little debugging

Reply:
@rickhanlonii do you have any ways to debug this ?

Reply:
Change
```
which yarn || curl -o- -L https://yarnpkg.com/install.sh | bash
``` 
to
```
which yarn || curl -o- -L https://yarnpkg.com/install.sh | bash && bash
```
It will start new shell where the previously installed `yarn` will be available.

Tested on `markhobson/node-chrome` docker image.

Reply:
Thanks a lot @peterdanis, I'm not familiar so much with docker build. Look like it solved the installing issue

Reply:
@AhnpGit isn't working. Will have to find another solution.

Reply:
@peterdanis I think there is something related to profile as discussion in https://github.com/yarnpkg/yarn/issues/601

Reply:
Yes, solved it @peterdanis @rickhanlonii 👍

Reply:
If it's too troublesome, we could just install from aptitude or something. I thought this script would be straight forward - apparently not

Reply:
I have one question, why do we need the one yarn-install-no-sudo ? I saw configuration for node-6 test using yarn-install and it works well. I think if we replace test-browser with yarn-install it will work like node-6 pipeline

Reply:
There is no sudo in docker image used by `test-browser` build job, hence the "no-sudo" variant.

Reply:
@peterdanis @SimenB I made it work. It is an ugly hack that I need to export path again before executing yarn. Somehow when exporting path for in one command line, it won't be available to the other command lines